fighters/common/src/opff/physics.rs

```diff
@@ -172,10 +172,76 @@ unsafe fn grab_jump_refresh(boma: &mut BattleObjectModuleAccessor) {
     }
 }
 
+unsafe fn dash_energy(fighter: &mut L2CFighterCommon) {
```

Comment:
Moved energy application from Dash's end status -> opff. Consequently, energy gets applied 1 frame earlier when transitioning from Dash -> Turn or Dash -> Dash.
This is because order of operation every frame seems to be opff -> energy func -> main status loop. If we apply energy in main loop, it doesn't get applied until the next frame. If we apply it in opff, it successfully overrides the energy func and gets applied on the current frame.